packages/app-desktop/gui/UpdateNotification/style.scss

```
@@ -1,11 +1,108 @@
+				text-decoration-color: currentColor;
+			}
+		}
+	}
+
+	// Release notes flow as natural content, with a subtle left accent
+	> .update-notification-release-notes {
+		max-height: 200px;
+		overflow-y: auto;
+		overflow-x: hidden;
+		scrollbar-gutter: stable;
+		word-break: break-word;
```

Stylelint (17.6.0)

[error] Unexpected deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)